runtime/testy/runtime.py

```diff
@@ -155,7 +156,8 @@ def checkTimedOut(func, *args):
       time.sleep(max(nextCall - time.time(), 0))
       execCount += 1
 
-    badThingsQueue.put(BadThing(sys.exc_info(), "Process Ended", event=BAD_EVENTS.END_EVENT))
+    if not terminated:
```

Comment:
Note that this section of code is also touched by #113, which has basically passed code review. I think you guys should make an effort to get all changes merged into master asap, instead of dragging out the merge conflicts for a long time.